src/domain/accounts/services/src/services/account_service_impl.rs

```diff
@@ -129,6 +161,63 @@ impl AccountService for AccountServiceImpl {
         self.account_repo
             .search_accounts_by_name_pattern(name_pattern, filters, pagination)
     }
+
+    async fn create_account(
+        &self,
+        account_name: &odf::AccountName,
+        email: email_utils::Email,
+        password: String,
```

Comment:
Q: maybe allocate a new type (`nutype`) that will not allow empty password and length less than 8?